src/rules/no-loud-comments/index.js

```diff
@@ -0,0 +1,37 @@
+import { utils } from "stylelint";
+import { namespace } from "../../utils";
+
+export const ruleName = namespace("no-loud-comments");
+
+export const messages = utils.ruleMessages(ruleName, {
+  rejected: "Use // for comments instead of /*"
+});
+
+function rule(primary) {
+  return (root, result) => {
+    const validOptions = utils.validateOptions(result, ruleName, {
+      actual: primary
+    });
+
+    if (!validOptions) {
+      return;
+    }
+
+    root.walkComments(comment => {
+      if (isLoudComment(comment)) {
+        utils.report({
+          message: messages.rejected,
+          node: comment,
+          result,
+          ruleName
+        });
+      }
+    });
+  };
+}
+
+function isLoudComment(comment) {
+  return comment.source.input.css.includes("/*");
```

Comment:
Hmm.. wouln't a double slash comment that starts with `//*` be reported if `.includes()` is used?

Reply:
Great call! I added a new test case for this and changed from using `includes()` to a more powerful regex.